DockerHubfile

```diff
@@ -0,0 +1,32 @@
+#### Create Jar 
+FROM gradle:jdk8 as builder 
+ 
+COPY . /home/gradle/app 
+USER root
+RUN chown -R gradle:gradle /home/gradle/app
+
+USER gradle
+WORKDIR /home/gradle/app 
+ 
+RUN mkdir -p api/src/main/resources/META-INF 
+RUN echo "build.version=$(gradle -q printVersion)" > api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.number=${BUILD_NUMBER:=docker}" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.commit=$(git rev-parse HEAD)" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.date=$(date)" >> api/src/main/resources/META-INF/build-info.properties 
+ 
+RUN gradle installDist bootRepackage 
```

Comment:
do you need to installDist and bootRepackage?

Reply:
I don't know the difference :see_no_evil:

Reply:
`bootRepackage` is spring boot gradle plugin specific task which is not really needed in this case as `installDist` does all the job and installed distribution is used below